packages/core/src/components/fields/SchemaField.js

```diff
@@ -249,7 +249,8 @@ function SchemaFieldRender(props) {
   const FieldTemplate =
     uiSchema["ui:FieldTemplate"] || registry.FieldTemplate || DefaultTemplate;
   let idSchema = props.idSchema;
-  const schema = retrieveSchema(props.schema, rootSchema, formData);
+  var schema = retrieveSchema(props.schema, rootSchema, formData);
```

Comment:
I'm assuming all `const -> var` related changes are due to the original PR being a bit old. In any case, const is highly preferable to var, so please revert all of these changes.